### PR TITLE
New exercise: Reverse String

### PR DESCRIPTION
**Pull request by hiljusti**

### CODE REVIEW

File: config.json [hunk 1]
```diff
@@ -75,6 +75,19 @@
...
+      {
+        "slug": "reverse-string",
+        "name": "Reverse String",
+        "uuid": "5d28be8e-395e-4e2a-aab6-c5777bcb0668",
+        "practices": [
+          "stack-effect",
+          "strings"
+        ],
+        "prerequisites": [
+          "stack-effect"
```
Comment (ErikSchierboom) [inline config.json:87]: Shouldn't `strings` also be a prerequisite here? I think you have to know about strings before being able to reverse a string, right?

Comment (hiljusti) [reply config.json:87]: Hmm maybe I'm fuzzy on the idea of what "prerequisites" is supposed to mean - I was thinking someone could learn strings while doing this?

I'll add it in, I'm probably expecting too much

Comment (hiljusti) [reply config.json]: Adding and rebasing

Comment (ErikSchierboom) [reply config.json]: @hiljusti It sounds like you got prerequisites and practices the other way around :) Prequisites are the things the student _must_ be familiar with in order for the student to be able to solve the exercises. Practices are the things the exercise can be used for to practice certain concepts on. So e.g. an exercise could have `strings` as a prerequisites (e.g. as the input or output parameter is a string), but have `recursion` as a a practices concept as the student _could_ solve the exercise with recursion.

See https://github.com/exercism/docs/blob/main/building/tracks/config-json.md#practice-exercises

**Review by ErikSchierboom — APPROVED** [review]

**Pull request merged**